spec/models/ip_spec.rb

```diff
@@ -50,4 +50,32 @@
       it { is_expected.to be_available }
     end
   end
+
+  context 'when many IPs already exist for the same location' do
+    let(:location_1) do
+      Location.create!(
+        address: 'Momentum Centre, London',
+        postcode: 'SE10SX',
+        organisation_id: create(:organisation).id
+      )
+    end
+
+    let(:ip) do
+      Ip.new(location_id: location_1.id, address: "127.0.2.1")
+    end
+
+    before { 20.times { create(:ip, location: location_1) } }
```

Comment:
Struggling what this 20.times sets the spec up for.
Is it related to the timeout spec below?

Reply:
If we're gonna chop it, I can dodge this question :sweat_smile: